docs/hardware/devices/techo/buttons.mdx

```diff
@@ -1,4 +1,4 @@
----
+    263452---
```

Comment:
what is this number for?

Reply:
Actually had no idea! Must have been an accidental keypress I didn't notice